Frontend/kalvians_management_system-kms/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Remove default generated Readme by create-react-app and add your application component details here, you can also add details to install and run react app in local here.